app/assets/stylesheets/views/_browse.scss

```diff
@@ -1,35 +1,33 @@
-$gutter-one-sixth: $gutter / 6;
-
 // scss-lint:disable NestingDepth
 .browse {
   .browse-panes {
-    @extend %contain-floats;
-    padding-bottom: $gutter;
+    @include govuk-clearfix;
+    padding-bottom: govuk-spacing(6);
     position: relative;
 
-    @include media(tablet) {
-      padding-top: $gutter;
+    @include govuk-media-query(tablet) {
+      padding-top: govuk-spacing(6);
     }
 
     &.section {
       .root-pane {
         display: none; // hide on mobile
 
-        @include media(tablet) {
+        @include govuk-media-query(tablet) {
           display: block;
           float: right;
           width: 25%;
         }
       }
 
       .section-pane {
-        @include media(tablet) {
+        @include govuk-media-query(tablet) {
           float: right;
           width: 30%;
           margin-right: 45%;
         }
 
-        @include media(desktop) {
+        @include govuk-media-query(desktop) {
```

Comment:
Just want to check - the documentation for these media queries has them written like this: `@include govuk-media-query($from: desktop)` - will the above still work?